templates/views/views-view-grid.html.twig

```diff
@@ -26,16 +26,13 @@
  * @ingroup themeable
  */
 #}
-
-{{ attach_library('common_design/cd-grid') }}
-
 {%
   set classes = [
     'views-view-grid',
     options.alignment,
     'cols-' ~ options.columns,
     'clearfix',
-    'cd-grid cd-grid--' ~ options.columns ~ '-col',
+    options.alignment == 'horizontal' and automatic_width == false ? 'cd-grid cd-grid--' ~ options.columns ~ '-col',
```

Comment:
@left23 I think it should be `options.automatic_width` here and below, according to the [preprocess function](https://api.drupal.org/api/drupal/core%21modules%21views%21views.theme.inc/function/template_preprocess_views_view_grid/8.9.x).

Reply:
Thanks, I've updated that.